hazelcast-enterprise/Dockerfile

```diff
@@ -35,32 +39,44 @@ ENV HZ_HOME="${HZ_HOME}" \
     CLASSPATH="" \
     JAVA_OPTS=""
 
+LABEL name="Hazelcast IMDG Enterprise" \
+      maintainer="info@hazelcast.com" \
+      vendor="Hazelcast, Inc." \
+      version="8.1" \
+      release="${HZ_VERSION}" \
+      url="http://www.hazelcast.com" \
+      summary="Hazelcast IMDG Enterprise Image, certified to RHEL 8.1" \
+      description="Starts a standalone Hazelcast server instance to form a cluster based on Kubernetes discovery inside Openshift/Kubernetes"
+
 # Expose port
 EXPOSE 5701
 
 COPY *.sh *.yaml *.jar *.properties ${HZ_HOME}/
+COPY licenses /licenses
 
 # Install
-RUN echo "Installing new APK packages" \
-    && apk add --no-cache openjdk11-jre bash apr curl procps nss \
-    && echo "Installing old OpenSSL 1.0 from Alpine 3.8" \
-    && apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.8/main/ openssl=1.0.2u-r0 \
+RUN mkdir -p "${HZ_HOME}/lib" "$HZ_CP_MOUNT" \
+    && echo "Disabling subscription-manager plugin to prevent redundant logs" \
+    && sed -i 's/^enabled=.*/enabled=0/g' /etc/dnf/plugins/subscription-manager.conf \
+    && dnf config-manager --disable \
+    && dnf -y update-minimal --security --sec-severity=Important --sec-severity=Critical --setopt=tsflags=nodocs \
+    && echo "Installing new packages" \
+    && dnf -y --setopt=tsflags=nodocs install java-11-openjdk &> /dev/null \
     && echo "Downloading Hazelcast and related JARs" \
-    && mkdir "${HZ_HOME}/lib" \
     && cd "${HZ_HOME}/lib" \
-    && for JAR_URL in ${HAZELCAST_ALL_URL} ${CACHE_API_URL} ${PROMETHEUS_AGENT_URL} ${NETTY_URL} ${NETTY_TCNATIVE_URL} ${EUREKA_PLUGIN_URLS} ${LOG4J2_URLS}; do curl -sf -O -L ${JAR_URL} || exit $?; done \
+    && for JAR_URL in ${HAZELCAST_ALL_URL} ${CACHE_API_URL} ${PROMETHEUS_AGENT_URL} ${NETTY_URL} ${NETTY_TCNATIVE_URL} ${LOG4J2_URLS}; do curl -sf -O -L ${JAR_URL}; done \
```

Comment:
Do we plan to have the Eureka plugin in? I see you removed it from here but didn't remove `ARG EUREKA_PLUGIN_URLS`. I'm fine with removing from Hazelcast Enterprise image, but could you please make sure none of our customers uses Eureka atm?

Reply:
Actually, I didn't have such a plan for this PR, it just a copy paste error. I want to remove it with another PR but I think we  can do it now 👍 I will talk with support team.